src/feedback.js

```diff
@@ -321,5 +321,67 @@ export function initFeedbacks() {
 		}
 	}
 
+	if (this.data.apiVersion >= 6) {
+		const UNAVAILABLE_STATUS = 0
+		const CACHING_STATUS = 1
+
+		feedbacks.nextElementCaching = {
+			type: 'boolean',
+			name: 'Next element caching',
+			description: 'Indicates if the upcoming element is caching',
+			defaultStyle: {
+				bgcolor: red,
+			},
+			options: [],
+			callback: ({ options }) => {
+				if (this.data.playoutRunning && this.data.elementsStatuses[this.data.upcomingElementId] === CACHING_STATUS) {
+					return true
+				}
+			},
+		}
+		feedbacks.nextElementUnavailable = {
+			type: 'boolean',
+			name: 'Next element unavailable',
+			description: 'Indicates if the upcoming element is unavailable',
+			defaultStyle: {
+				bgcolor: yellow,
+			},
+			options: [],
+			callback: ({ options }) => {
+				if (this.data.playoutRunning && this.data.elementsStatuses[this.data.upcomingElementId] === UNAVAILABLE_STATUS) {
+					return true
+				}
+			},
+		}
+		feedbacks.templateInsertStatus = {
+			type: 'boolean',
+			name: 'Status of the recent template insert',
+			description: 'Indicates if a triggered template insertion is running',
```

Comment:
I think "...is running" does not describe both of your options is running/has failed